Plugins/Wox.Plugin.Sys/Main.cs

```diff
@@ -56,12 +56,21 @@ public List<Result> Query(Query query)
             var results = new List<Result>();
             foreach (var c in commands)
             {
-                var titleScore = StringMatcher.FuzzySearch(query.Search, c.Title).ScoreAfterSearchPrecisionFilter();
-                var subTitleScore = StringMatcher.FuzzySearch(query.Search, c.SubTitle).ScoreAfterSearchPrecisionFilter();
-                var score = Math.Max(titleScore, subTitleScore);
+                var titleMatch = StringMatcher.FuzzySearch(query.Search, c.Title);
```

Comment:
@jjw24 Maybe we can add something more general here?
as it seems that this code can return in many places?